app/forms/finance/ecf/change_training_status_form.rb

```diff
@@ -34,16 +34,20 @@ def save
         return false unless valid?
         return true if status_unchanged?
 
-        klass = "Participants::#{action_class_name}::#{participant_class_name}".constantize
-        klass.call(
-          params: {
-            cpd_lead_provider:,
-            course_identifier:,
-            participant_id: participant_profile.participant_identity.external_identifier,
-            reason:,
-            force_training_status_change: true,
-          },
-        )
+        params = {
+          cpd_lead_provider:,
+          course_identifier:,
+          participant_id: participant_profile.participant_identity.external_identifier,
+        }
+
+        if training_status == "active"
```

Comment:
need to rethink this when other refactor prs are merged, since we are no blocked with testing